pkg/porter/install_test.go

```diff
@@ -89,6 +90,7 @@ func TestInstallOptions_validateParams(t *testing.T) {
 		sharedOptions{
 			Params: []string{"A=1", "B=2"},
 		},
+		BundlePullOptions{},
```

Comment:
Since you declared InstallOptions with BundlePullOptions as an embedded type (and not a pointer), you don't need to have this line here. Same for the other tests below. I

The only time you need stuff like this line here is if you defined it like this instead

```
type InstallOptions struct {
  *BundlePullOptions
}
```

Reply:
huh, the compiler complained until I added this.